dashboard/config/blocks/GamelabJr/gamelab_setForegroundEffect.json

```diff
@@ -0,0 +1,47 @@
+{
+  "category": "World",
+  "config": {
+    "func": "setForegroundEffect",
+    "blockText": "do {EFFECT} effect",
+    "args": [
+      {
+        "name": "EFFECT",
+        "options": [
+          [
+            "bubbles",
+            "\"bubbles\""
+          ],
+          [
+            "confetti",
+            "\"confetti\""
+          ],
+          [
+            "hearts",
+            "\"hearts\""
+          ],
+          [
+            "happy emojis",
+            "\"emojis\""
+          ],
+          [
+            "rain",
+            "\"rain\""
+          ],
+          [
+            "starburst",
+            "\"starburst\""
+          ],
+          [
+            "shattering glass",
+            "\"glass\""
+          ],
+          [
+            "twinkling",
+            "\"twinkling\""
+          ]
+        ]
+      }
+    ],
+    "style": "world_blocks"
+  }
+}
```

Comment:
Direct copy of https://github.com/code-dot-org/code-dot-org/blob/mike/effects-for-spritelab/dashboard/config/blocks/Poetry/Poetry_setForegroundEffect.json

The old block cannot be deleted because students already have `Poetry_setForegroundEffect` in their projects and level progress.